contracts/contracts/mcms/rbac_timelock.tolk

```diff
@@ -1368,7 +1370,17 @@ fun Timelock<T>.hookTrait__AccessControl(self): AccessControl<T>{
 // --- Getters ---
 
 get fun typeAndVersion(): (slice, slice) {
-     return ("com.chainlink.ton.mcms.Timelock", "1.0.0");
+    return ("com.chainlink.ton.mcms.Timelock", CONTRACT_VERSION);
+}
+
+// Returns the current code of the contract.
+get fun code(): cell {
+    return contract.getCode();
+}
+
+// Returns the sha256 hash of the current code of the contract.
+get fun codeHash(): int {
```

Comment:
Why do we now require adding these getters to all contracts? Isn't this redundant functionality to be hosted in the contracts and the info (code) can easily be sourced by a client (form an compiled artifact or address)?